bin/numa_prefix

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env perl
```

Comment:
Curious... why perl?

Reply:
because i know perl and it exists everywhere usually

Reply:
We already assume python is installed and your other script is written in python. I think this script should also be written in python.

Reply:
i will keep this perl/python discussion civil and make the change to python

Reply:
As much as I dislike perl the one thing it _is_ better at than python is regexes. Is this too complicated to be a few `sed` commands?

Reply:
btw, perl is used within the rocket-chip generator, so its safe to say perl is required for chipyard anyways. so i am lowering my white flag on this

yes, its too complicated for sed.

Reply:
*too complicated for readable sed

Reply:
I think the majority if not everyone here uses `python` so I would recommend this be ported to `python` if you potentially want others to help with the upkeep (if it breaks). However, I think this is small enough that `perl` is fine. With that being said, I would like some more comments within the code.

Reply:
i'm going to comment the current perl file a bit for clarity and open an enhancement to convert it to python.